### DISCUSSION TITLE
Feature wishlist

### DISCUSSION THREAD

**Discussion by Akselmo** [DroidWorksStudio/EasyLauncher#69] [Ideas]
Before I begin, I want to preface that I know not every feature can make it and that is totally fine! But here's some items I'd love to see eventually. Some of them are what mLauncher already has.

- [ ] Wallpaper opacity setting, like in mLauncher, helps a lot with readability
- [ ] Text shadows, also helps with readability
    - Could be a customizable color? Like other colors
- [ ] Follow android system font
    - On my android 14 device I can change the system font to Lato, Rubik and some others.
    - Would be cool if the launcher would follow this font setting!
    - This is in the wallpaper and style settings in my device
    - Could also just add a list of fonts that are installed with the app and select from those
- [ ] Custom icon pack support
- [ ] Calendar widget
- [ ] Filter strength setting to search, like in mLauncher
    - I tend to tone this down a bit usually

Again, I understand if not all of these can make it but the wallpaper opacity setting would be very good to have.

There's lots of potential in EasyLauncher and I am excited to see how it develops! I am currently now using mLauncher as daily driver but I do try out every EasyLauncher release.

Thanks for the great apps!

**Comment by CreativeCodeCat** [comment]
Ya I was looking at adding some of them features into EasyLauncher as it's under 2 months old compared to mLauncher that's had 3 years work on it I have tons of work to still do with EasyLauncher

**Comment by ddogfoodd** [comment]
Hey thanks for the great launcher!!

I would like to add to the wishlist:
Dynamic Spacing between Clock/date/wotd/battery and favorite app areas.

Like when disabling the any of those clock/date/wotd/battery currently there will be empty space, it would be nice to use this empty space for the favorite apps instead.

**Reply by CreativeCodeCat** [reply]
That sounds like a good idea I'll look at doing that before I release 0.2.0 at the end of the week.

**Comment by ddogfoodd** [comment]
Is support for app widgets planned or is this out of this launchers scope?
